File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectEntryCommand_View.cpp

#include "SelectEntryCommand_View.h"
#include "SelectEntryCommand_View.h"
#include <vector>
#include "SelectEntryCommand_ViewMyOptionsOption.h"
#include <string>

namespace commands
{
  std::vector<std::string> SelectEntryCommand_View::getMyElementsComboBoxEntries( ) 
  {
    return this->MyElementsComboBoxEntries;
  }
  std::string SelectEntryCommand_View::getMyElementsComboBoxSelectedEntry( ) 
  {
    return this->MyElementsComboBoxSelectedEntry;
  }
  SelectEntryCommand_ViewMyOptionsOption SelectEntryCommand_View::getMyOptionsRadioButtonGroupSelectedEntry( ) 
  {
    return this->MyOptionsRadioButtonGroupSelectedEntry;
  }
  void SelectEntryCommand_View::myElementsEntrySelected(std::string entryName ) 
  {
    
  }
  void SelectEntryCommand_View::myOptionsEntrySelected(SelectEntryCommand_ViewMyOptionsOption entryName ) 
  {
    
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectEntryCommand_View.cs
/// <filename>
///     SelectEntryCommand_View.cs
/// </filename>
using System.Collections.Generic;
using commands;

namespace commands
{
    internal class SelectEntryCommand_View
  {
        private System.Collections.Generic.List<string> MyElementsComboBoxEntries = new System.Collections.Generic.List<string>();
     
        public     virtual System.Collections.Generic.List<string> getMyElementsComboBoxEntries()
    {
      return this.MyElementsComboBoxEntries;
    }
     
        private string MyElementsComboBoxSelectedEntry;
     
        public     virtual string getMyElementsComboBoxSelectedEntry()
    {
      return this.MyElementsComboBoxSelectedEntry;
    }
     
        private commands.SelectEntryCommand_ViewMyOptionsOption MyOptionsRadioButtonGroupSelectedEntry;
     
        public     virtual commands.SelectEntryCommand_ViewMyOptionsOption getMyOptionsRadioButtonGroupSelectedEntry()
    {
      return this.MyOptionsRadioButtonGroupSelectedEntry;
    }
     
        public     virtual void myElementsEntrySelected(string entryName)
    {
      
    }
     
        public     virtual void myOptionsEntrySelected(commands.SelectEntryCommand_ViewMyOptionsOption entryName)
    {
      
    }
  }
}

